### ISSUE TITLE
Remove exceptiongroups from getting started page?

### ISSUE THREAD

**Issue by The-Compiler** [pytest-dev/pytest#12883]
Motivated by #12880, which proposes adding even more content to the "getting started with pytest" page, intended for teaching  the very basics of pytest's functionality:

Personally I think exception groups shouldn't be on a [page describing how to write your first test](https://docs.pytest.org/en/latest/getting-started.html) at all. It's not a feature new pytest (and potentially Python) users necessarily need to know about when just starting out. We don't even explain e.g. `match=` for `pytest.raises` on that page.

If we keep it, at the very least it shouldn't be between an example being shown, and that example being run later...:

![image](https://github.com/user-attachments/assets/e6ee8f80-c2a6-42f1-8792-fe92cf34478a)

But IMHO it has no place there at all, and the [How to write and report assertions in tests](https://docs.pytest.org/en/latest/how-to/assert.html#matching-exception-groups) page already has a very similar explanation, where it does seem to fit in well.

@lanzz @Zac-HD what do you think?

**Comment by lanzz** [comment]
I don't mind removing it from `getting-started.rst`, your reasoning is compelling and I might have gone a bit overboard in adding it there in the first place.

**Comment by Zac-HD** [comment]
Sounds good to me - let's cut back to the first very simple example, and then just say e.g. "[Assertions about expected exceptions](https://docs.pytest.org/en/latest/how-to/assert.html#assertions-about-expected-exceptions) explains how to match exception messages or the contents of `ExceptionGroup`." so that it's easy to find for anyone who knows they need that.

**Comment by jakkdl** [comment]
Agree, it can link to https://pytest--13192.org.readthedocs.build/en/13192/how-to/assert.html#assertions-about-expected-exceptions for more details. I'll make some changes in #13192
